.github/workflows/optimize-ci.yml

```diff
@@ -59,6 +59,128 @@ jobs:
     with:
       branch: ${{ github.head_ref }}
 
+  es-schema-integrity-test:
```

Comment:
One more thought: do we want to deploy artifacts even when this fails or should it be added to the requirements? Same also for the rerun step

Reply:
Good question, and I think the answer is yes to both. Added